src/photos/ducks/backup/components/BackupHeader.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import { Icon } from 'cozy-ui/transpiled/react'
+import PhoneUploadIcon from 'cozy-ui/transpiled/react/Icons/PhoneUpload'
+import palette from 'cozy-ui/transpiled/react/palette'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+const BackupHeader = () => {
+  const { t } = useI18n()
+
+  return (
+    <div className="u-flex u-flex-column u-flex-items-center">
+      <Icon icon={PhoneUploadIcon} size="64" color={palette.slateGrey} />
```

Comment:
Are we sure to use this color directly? Without any semantics?

Reply:
I changed it by _var(--iconTextColor)_ which is way better